packages/appkit/src/client.ts

```diff
@@ -1750,12 +1695,12 @@ export class AppKit {
     return this.chainAdapters?.[namespace]
   }
 
+  private isSafe() {
+    return typeof window !== 'undefined'
+  }
```

Comment:
we have an util for this https://github.com/reown-com/appkit/blob/main/packages/core/src/utils/CoreHelperUtil.ts#L38-L40